Solutions/MCBSTM32F400/netmfdbg/netmfdbg.uvoptx

```diff
@@ -192,7 +192,56 @@
           <Name>UL2CM3(-S0 -C0 -P0 -FD20000000 -FC1000 -FN1 -FF0STM32F4xx_1024 -FS08000000 -FL0100000 -FP0($$Device:STM32F407IGHx$CMSIS\Flash\STM32F4xx_1024.FLM))</Name>
         </SetRegEntry>
       </TargetDriverDllRegistry>
-      <Breakpoint/>
+      <Breakpoint>
```

Comment:
That's how uVision does it.For the most part the paths are hints and it can resolve things, but we can't really control what it stores in these files and their contents are completely undocumented.